### PR TITLE
Added script to perform Canny-Edge-detection

### PR DESCRIPTION
**Pull request by Namyalg**
This script takes the path of an image as a command line argument and displays the image after applying the Canny filter and saves a copy in the current folder of the image 

PR relates to #24

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Nice work @Namyalg !

# Suugestions:

- Try to integrate progress bars for the time spent in the overall operations using the tqdm library
- Can you please replace the sample image with the default [taxi one] we are using for this repo and update the readme

**Comment by Namyalg** [comment]
Okay, I will do that

**Comment by Namyalg** [comment]
@smaranjitghose , I have made the changes as suggested above!

File: ty.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from tqdm import tqdm 
+
```
Comment (smaranjitghose) [inline ty.py]: What is this file for?

Comment (Namyalg) [reply ty.py]: I have deleted it, I forgot to delete it earlier. My apologies

**Comment by smaranjitghose** [comment]
@Namyalg Can you please resolve the merge confilicts

**Comment by Namyalg** [comment]
I have resolved the conflicts @smaranjitghose

**Pull request merged**